packages/custom-elements/src/native-shim.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+import {} from './Externs';
```

Comment:
I'm not sure it makes sense to convert the native-shim to a module given that its purpose is to be used in scenarios where you can't use modules. We need to make sure this is output as ES5 and I don't think that the gulpfile from master did anything with native-shim.js so I don't think that anything is happening to make this file work as expected currently.

Reply:
This was a type-only import, so it was elided during emission. Once TS 3.9 comes we can make that explicit with `import type`.

As it turned out though, this was unnecessary, dropped.